tests/sender/send_data.py

```diff
@@ -75,6 +76,19 @@ def test_compose_mem(self):
                          b'<14>1991-02-20 12:00:00 %s test.tag: '
                          % self.localhost.encode("utf-8"))
 
+    @staticmethod
+    def read(con, length: int):
+        if not select.select([con.socket], [], [], con.socket_timeout)[0]:
+            raise Exception("Timeout reached during read operation")
```

Comment:
A generic exception here? Is there any other more concrete type to apply?

Reply:
Yes, it is just a test case